<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta name="google-site-verification" content=""> <meta http-equiv="Permissions-Policy" content="interest-cohort=()"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Mike Nsubuga</title> <meta name="author" content="Mike Nsubuga"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="academic-webiste, mike-nsubuga, nsubuga"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://mikensubuga.github.io/publications/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Mike </span>Nsubuga</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <div class="publications"> <p>An up-to-date list is available on <a href="https://scholar.google.com/citations?hl=en&amp;user=czbZn2IAAAAJ&amp;hl" rel="external nofollow noopener" target="_blank">Google Scholar</a>.</p> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Nat Protoc</abbr></div> <div id="lujumba_practical_2025" class="col-sm-8"> <div class="title">A practical guide to identifying associations between tandem repeats and complex human traits using consensus genotypes from multiple tools</div> <div class="author"> Ibra Lujumba, Yagoub Adam, Helyaneh Ziaei Jam, and <span class="more-authors" title="click to view 24 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '24 more authors' ? 'Itunuoluwa Isewon, Nomakhosazana Monnakgotla, Yang Li, Blessing Onyido, Kakembo Fredrick, Faith Adegoke, Jerry Emmanuel, Jumoke Adeyemi, Olajumoke Ibitoye, Samuel Owusu-Ansah, Matthew Boladele Akanle, Habi Joseph, Mike Nsubuga, Ronald Galiwango, Martin Okitwi, Namuswe Magdalene, Odur Walter, Zama Mngadi, Marion Adebiyi, Jelili Oyelade, Melissa Nel, Daudi Jjingo, Melissa Gymrek, Ezekiel Adebiyi' : '24 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">24 more authors</span> </div> <div class="periodical"> <em>Nature Protocols</em>, Sep 2025 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41596-025-01231-y"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41596-025-01231-y" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Tandem repeats (TRs) are highly variable loci in the human genome that are linked to various human phenotypes. Accurate and reliable genotyping of TRs is important in understanding population TR variation dynamics and their effects in TR-trait association studies. In this protocol, we describe how to generate high-quality consensus TR genotypes for population genomics studies. In particular, we detail steps to: (i) perform TR genotyping from short-read whole-genome sequencing data by using the HipSTR, GangSTR, adVNTR and ExpansionHunter tools, (ii) perform quality control checks on TR genotypes by using TRTools and (iii) integrate TR genotypes from different tools by using EnsembleTR. We further discuss how to visualize and investigate TR variation patterns to identify population-specific expansions and perform TR-trait association analyses. We demonstrate the utility of these steps by analyzing a small dataset from the 1000 Genomes Project. In addition, we recapitulate a previously identified association between TR length and gene expression in the African population and provide a generalized discussion on TR analysis and its relevance to identifying complex traits. The expected time for installing the necessary software for each section is ~10 min. The expected run time on the user’s desired dataset can vary from hours to days depending on factors such as the size of the data, input parameters and the capacity of the computing infrastructure.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Nat Comms</abbr></div> <div id="gregorova_early_2025" class="col-sm-8"> <div class="title">Early NK-cell and T-cell dysfunction marks progression to severe dengue in patients with obesity and healthy weight</div> <div class="author"> Michaela Gregorova, Marianna Santopaolo, Lucy C. Garner, and <span class="more-authors" title="click to view 20 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '20 more authors' ? 'Rahma F. Hayati, Divya Diamond, Narayan Ramamurthy, Vi Thuy Tran, Nguyet Minh Nguyen, Kate J. Heesom, Vuong Lam Nguyen, Eben Jones, Mike Nsubuga, Curtis Luscombe, Hoa Thi My Vo, Chanh Quang Ho, Chau Thi Xuan Nguyen, Tam Thi Hoai Dong, Duyen Thi Le Huynh, Tam Thi Cao, Andrew D. Davidson, Paul Klenerman, Sophie Yacoub, Laura Rivino' : '20 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">20 more authors</span> </div> <div class="periodical"> <em>Nature Communications</em>, Jul 2025 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41467-025-60941-9"></span> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-025-60941-9" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Dengue is a mosquito-borne virus infection affecting half of the world’s population for which therapies are lacking. The role of T and NK-cells in protection/immunopathogenesis remains unclear for dengue. We performed a longitudinal phenotypic, functional and transcriptional analyses of T and NK-cells in 124 dengue patients using flow cytometry and single-cell RNA-sequencing. We show that T/NK-cell signatures early in infection discriminate patients who develop severe dengue (SD) from those who do not. These signatures are exacerbated in patients with overweight/obesity compared to healthy weight patients, supporting their increased susceptibility to SD. In SD, CD4+/CD8+ T-cells and NK-cells display increased co-inhibitory receptor expression and decreased cytotoxic potential compared to non-SD. Using transcriptional and proteomics approaches we show decreased type-I Interferon responses in SD, suggesting defective innate immunity may underlie NK/T-cell dysfunction. We propose that dysfunctional T and NK-cell signatures underpin dengue pathogenesis and may represent novel targets for immunomodulatory therapy in dengue.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BJS</abbr></div> <div id="please_462_2025" class="col-sm-8"> <div class="title">462 Powering Precision Medical Education Through AI-Generated Country-Specific Global Surgery Podcasts: A Low-or-Middle-Income-Country Case Study</div> <div class="author"> H Please, M Nsubuga, T M Kintu, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'A Bakesiga, K Stewart, S Navarro' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>BJS</em>, Jun 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1093/bjs/znaf128.052"></span> <span class="__dimensions_badge_embed__" data-doi="10.1093/bjs/znaf128.052" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Develop an AI-generated Global Surgery podcast series tailored to healthcare professionals’ needs in specific countries, focussing on Uganda as a pilot. This precision education model addresses disparities in global surgical education, historically driven by high-income countries (HICs). Podcasts were chosen for their scalability, accessibility, and self-directed learning.Building on prior data from our international trainee-based needs assessment (145 survey respondents from 18 countries; 94 from Low-and-Middle-Income-Countries, LMICs; 51 from High-Income-Countries, HICs). Generative AI (ChatGPT 4.0, OpenAI) and prompt engineering were used to develop a global surgery curriculum from survey data from Ugandan respondents (selected because Uganda had one of the highest response rates, n=31). The resultant curriculum was then adapted into a podcast format featuring country-specific themes using NotebookLM (Google), using a format of exercises, case studies, and expert interviews.The preliminary outputs of the survey and podcasts highlight diverse country-specific priorities; for HICs these include access to resources, research opportunities, and infrastructure development; for LMICs they emphasise the burden of surgical disease, use of local case studies, and foundational concepts in global surgery.By tailoring content to regional needs and leveraging storytelling to foster engagement, podcasts can complement traditional curricula to address gaps in access and relevance. Limitations include current AI-generative software offering only US accents, which may not suit learners elsewhere, reflecting concerns about HICs bias in AI-generated content. Future research will validate the podcasts through focus groups with Ugandan surgical trainees, and explore speech-processing for regional accents, or blending human voice with AI-generated transcripts.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BMC</abbr></div> <div id="nsubuga_enhancing_2025" class="col-sm-8"> <div class="title">Enhancing trauma triage in low-resource settings using machine learning: a performance comparison with the Kampala Trauma Score</div> <div class="author"> Mike Nsubuga, Timothy Mwanje Kintu, Helen Please, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Kelsey Stewart, Sergio M. Navarro' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>BMC Emergency Medicine</em>, Jan 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1186/s12873-025-01175-2"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12873-025-01175-2" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Traumatic injuries are a leading cause of morbidity and mortality globally, with a disproportionate impact on populations in low- and middle-income countries (LMICs). The Kampala Trauma Score (KTS) is frequently used for triage in these settings, though its predictive accuracy remains under debate. This study evaluates the effectiveness of machine learning (ML) models in predicting triage decisions and compares their performance to the KTS.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BMC</abbr></div> <div id="nabisubi_beyond_2025" class="col-sm-8"> <div class="title">Beyond the fever: shotgun metagenomic sequencing of stool unveils pathogenic players in HIV-infected children with non-malarial febrile illness</div> <div class="author"> Patricia Nabisubi, Stephen Kanyerezi, Grace Kebirungi, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Ivan Sserwadda, Mike Nsubuga, Grace Kisitu, Patricia Ntege Nahirya, Bonny Mulindwa, George P. Akabwai, Sylvia Nantongo, Adeodata Kekitiinwa, Edgar Kigozi, Nsubuga Moses Luutu, Fred Ashaba Katabazi, Leymon Kalema, Andrew Katabalwa, Daudi Jjingo, Gerald Mboowa' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>BMC Infectious Diseases</em>, Jan 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1186/s12879-025-10517-1"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12879-025-10517-1" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Non-malarial febrile illnesses (NMFI) pose significant challenges in HIV-infected children, often leading to severe complications and increased morbidity. While traditional diagnostic approaches focus on specific pathogens, shotgun metagenomic sequencing offers a comprehensive tool to explore the microbial landscape underlying NMFI in this vulnerable population ensuring effective management.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">IHTP</abbr></div> <div id="bolton_accelerating_2025" class="col-sm-8"> <div class="title">Accelerating innovation by integrating artificial intelligence into a global surgery hackathon</div> <div class="author"> William Bolton, Priyansh Nathani, Helen Please, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Johnny Philomen, Nadhirah Abd Kahar, Noel Aruparayil, Soham Bandyopadhyay, Vijna Boodhoo, Michael Magoha, Mike Nsubuga, Anurag Mishra, Pankaj Jani, Joshua Burke, Ryan Mathew, Peter Culmer' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>International Health Trends and Perspectives</em>, Mar 2025 </div> <div class="periodical"> Number: 1 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.32920/ihtp.v5i1.2332"></span> <span class="__dimensions_badge_embed__" data-doi="10.32920/ihtp.v5i1.2332" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Background: Investigating in AI’s utility in diverse learning environments can provide insights into its broader applicability in healthcare innovation. Objectives: This study aims to assess the impact on idea generation and implications of incorporating generative AI technology into the framework of a global surgery hackathon, focusing on its use with surgical care providers in Sub-Saharan Africa. Method: A 120-minute interdisciplinary hackathon in Kenya was organised. The event featured the use of ChatGPT, a large language AI model from OpenAI, to facilitate and guide team discussions and solution development. Data was collected through direct observations and discussions among participants. Results and Conclusions: The hackathon saw active participation from ninety attendees, who were divided into ten teams of 8-12 members each. These groups utilised AI to seek information, derive inspiration, and refine their ideas. Notable challenges identified included issues related to AI-generated biases and the accuracy of information provided. The study serves as a proof-of-concept that generative AI can effectively be integrated into hackathons to foster innovation, with the caveat that future implementations should focus on developing unbiased and accurate AI models. This approach has significant potential to improve educational strategies and operational efficiency in the healthcare and technology sectors.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JCTS</abbr></div> <div id="kintu_369_2025" class="col-sm-8"> <div class="title">369 Precision education and generative AI in surgery utilization study: A framework for global surgical education</div> <div class="author"> Timothy Kintu, Mike Nsubuga, Allan Bakesiga, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Helen Please, Kelsey Stewart, Sergio M. Navarro' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Journal of Clinical and Translational Science</em>, Apr 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1017/cts.2024.994"></span> <span class="__dimensions_badge_embed__" data-doi="10.1017/cts.2024.994" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Objectives/Goals: Global surgical education is largely driven by high-income countries (HICs), with curricula not tailored to the needs of low- and middle-income countries (LMICs). This study assessed country-specific needs for global surgical curricula and used generative AI to develop tailored curricula. Methods/Study Population: A curriculum framework was developed using expert opinion. Using a focused needs assessment survey, we evaluated international medical students’ and trainees’ needs for structured global surgery curricula, covering research, education, data and develop tailored curriculum templates for each country, ensuring alignment with the distinct needs of respective LMIC and HIC respondents. The AI-generated curricula were then compared across countries to identify variations in content and focus areas. Results/Anticipated Results: A total of 145 respondents from 18 countries and 6 continents participated, with 94 from LMICs and 51 from HICs. Four countries [Uganda (n = 31), Nigeria (n = 34), the USA (n = 23), and the UK (n = 23)] had more than 10 respondents, with the creation of a country specific global surgery curriculum. Curricula developed by HIC trainees focused on access to resources and infrastructure, future directions of global surgical research, and the role of medical students and early career development with a decreased focus on the history of global surgery. LMIC country-based curriculum focused on introducing the concepts of global surgery, quantifying the burden and epidemiology of surgical disease and had a greater emphasis on case studies and use cases, with decreased focus on resources and collaboration. Discussion/Significance of Impact: The research introduces a “precision education” approach that could help close the surgical education access gap globally. Further pilot and qualitative studies are necessary to validate the feasibility of AI-generated needs-based curricula.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BMC</abbr></div> <div id="nsubuga_generalizability_2024" class="col-sm-8"> <div class="title">Generalizability of machine learning in predicting antimicrobial resistance in E. coli: a multi-country case study in Africa</div> <div class="author"> Mike Nsubuga, Ronald Galiwango, Daudi Jjingo, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Gerald Mboowa' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>BMC Genomics</em>, Mar 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1186/s12864-024-10214-4"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12864-024-10214-4" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Antimicrobial resistance (AMR) remains a significant global health threat particularly impacting low- and middle-income countries (LMICs). These regions often grapple with limited healthcare resources and access to advanced diagnostic tools. Consequently, there is a pressing need for innovative approaches that can enhance AMR surveillance and management. Machine learning (ML) though underutilized in these settings, presents a promising avenue. This study leverages ML models trained on whole-genome sequencing data from England, where such data is more readily available, to predict AMR in E. coli, targeting key antibiotics such as ciprofloxacin, ampicillin, and cefotaxime. A crucial part of our work involved the validation of these models using an independent dataset from Africa, specifically from Uganda, Nigeria, and Tanzania, to ascertain their applicability and effectiveness in LMICs.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BMC</abbr></div> <div id="babirye_machine_2024" class="col-sm-8"> <div class="title">Machine learning-based prediction of antibiotic resistance in Mycobacterium tuberculosis clinical isolates from Uganda</div> <div class="author"> Sandra Ruth Babirye, Mike Nsubuga, Gerald Mboowa, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Charles Batte, Ronald Galiwango, David Patrick Kateete' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>BMC Infectious Diseases</em>, Dec 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1186/s12879-024-10282-7"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12879-024-10282-7" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Efforts toward tuberculosis management and control are challenged by the emergence of Mycobacterium tuberculosis (MTB) resistance to existing anti-TB drugs. This study aimed to explore the potential of machine learning algorithms in predicting drug resistance of four anti-TB drugs (rifampicin, isoniazid, streptomycin, and ethambutol) in MTB using whole-genome sequence and clinical data from Uganda. We also assessed the model’s generalizability on another dataset from South Africa.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BMJ</abbr></div> <div id="please_virtual_2024" class="col-sm-8"> <div class="title">Virtual reality technology for surgical learning: qualitative outcomes of the first virtual reality training course for emergency and essential surgery delivered by a UK–Uganda partnership</div> <div class="author"> Helen Please, Karamveer Narang, William Bolton, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Mike Nsubuga, Henry Luweesi, Ndiwalana Billy Richards, John Dalton, Catherine Tendo, Mansoor Khan, Daudi Jjingo, Mahmood F. Bhutta, Dimitra Petrakaki, Jagtar Dhanda' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>BMJ Open Quality</em>, Jan 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1136/bmjoq-2023-002477"></span> <span class="__dimensions_badge_embed__" data-doi="10.1136/bmjoq-2023-002477" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Introduction The extensive resources needed to train surgeons and maintain skill levels in low-income and middle-income countries (LMICs) are limited and confined to urban settings. Surgical education of remote/rural doctors is, therefore, paramount. Virtual reality (VR) has the potential to disseminate surgical knowledge and skill development at low costs. This study presents the outcomes of the first VR-enhanced surgical training course, ‘Global Virtual Reality in Medicine and Surgery’, developed through UK-Ugandan collaborations.Methods A mixed-method approach (survey and semistructured interviews) evaluated the clinical impact and barriers of VR-enhanced training. Course content focused on essential skills relevant to Uganda (general surgery, obstetrics, trauma); delivered through: (1) hands-on cadaveric training in Brighton (scholarships for LMIC doctors) filmed in 360°; (2) virtual training in Kampala (live-stream via low-cost headsets combined with smartphones) and (3) remote virtual training (live-stream via smartphone/laptop/headset).Results High numbers of scholarship applicants (n=130); registrants (Kampala n=80; remote n=1680); and attendees (Kampala n=79; remote n=556, 25 countries), demonstrates widespread appetite for VR-enhanced surgical education. Qualitative analysis identified three key themes: clinical education and skill development limitations in East Africa; the potential of VR to address some of these via 360° visualisation enabling a ‘knowing as seeing’ mechanism; unresolved challenges regarding accessibility and acceptability.Conclusion Outcomes from our first global VR-enhanced essential surgical training course demonstrating dissemination of surgical skills resources in an LMIC context where such opportunities are scarce. The benefits identified included environmental improvements, cross-cultural knowledge sharing, scalability and connectivity. Our process of programme design demonstrates that collaboration across high-income and LMICs is vital to provide locally relevant training. Our data add to growing evidence of extended reality technologies transforming surgery, although several barriers remain. We have successfully demonstrated that VR can be used to upscale postgraduate surgical education, affirming its potential in healthcare capacity building throughout Africa, Europe and beyond.Data may be obtained from a third party and are not publicly available. The participants of this study did not give consent for their data to be shared publicly, so the empirical data are not available due to confidentiality reasons.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">F1000Research</abbr></div> <div id="mboowa_rise_2024" class="col-sm-8"> <div class="title">The rise of pathogen genomics in Africa</div> <div class="author"> Gerald Mboowa, Francis Kakooza, Moses Egesa, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Stephen Tukwasibwe, Stephen Kanyerezi, Ivan Sserwadda, Benson R. Kidenya, Jupiter Marina Kabahita, Maria Magdalene Namaganda, Mike Nsubuga, Patricia Nabisubi, Alisen Ayitewala, Grace Kebirungi, Esther Nakafu, Natasha Patience Akwii' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> May 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.12688/f1000research.147114.1"></span> <span class="__dimensions_badge_embed__" data-doi="10.12688/f1000research.147114.1" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The routine genomic surveillance of pathogens in diverse geographical settings and equitable data sharing are critical to inform effective infection control and therapeutic development. The coronavirus disease 2019 (COVID-19) pandemic highlighted the importance of routine genomic surveillance of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) to detect emerging variants of concern. However, the majority of high-income countries sequenced &gt;0.5% of their COVID-19 cases, unlike low- and middle-income countries. By the end of 2022, many countries around the world had managed to establish capacity for pathogen genomic surveillance. Notably, Beta and Omicron; 2 of the 5 current SARS-CoV-2 variants of concern were first discovered in Africa through an aggressive sequencing campaign led by African scientists. To sustain such infrastructure and expertise beyond this pandemic, other endemic pathogens should leverage this investment. Therefore, countries are establishing multi-pathogen genomic surveillance strategies. Here we provide a catalog of the current landscape of sequenced and publicly shared pathogens in different countries in Africa. Drawing upon our collective knowledge and expertise, we review the ever-evolving challenges and propose innovative recommendations.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BMC</abbr></div> <div id="nsubuga_ugandan_2024" class="col-sm-8"> <div class="title">The Ugandan sickle Pan-African research consortium registry: design, development, and lessons</div> <div class="author"> Mike Nsubuga, Henry Mutegeki, Daudi Jjingo, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Deogratias Munube, Ruth Namazzi, Robert Opoka, Philip Kasirye, Grace Ndeezi, Heather Hume, Ezekiel Mupere, Grace Kebirungi, Isaac Birungi, Jack Morrice, Mario Jonas, Victoria Nembaware, Ambroise Wonkam, Julie Makani, Sarah Kiguli' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>BMC Medical Informatics and Decision Making</em>, Jul 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1186/s12911-024-02618-9"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12911-024-02618-9" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Sub-Saharan Africa bears the highest burden of sickle cell disease (SCD) globally with Nigeria, Democratic Republic of Congo, Tanzania, Uganda being the most affected countries. Uganda reports approximately 20,000 SCD births annually, constituting 6.67% of reported global SCD births. Despite this, there is a paucity of comprehensive data on SCD from the African continent. SCD registries offer a promising avenue for conducting prospective studies, elucidating disease severity patterns, and evaluating the intricate interplay of social, environmental, and genetic factors. This paper describes the establishment of the Sickle Pan Africa Research Consortium (SPARCo) Uganda registry, encompassing its design, development, data collection, and key insights learned, aligning with collaborative efforts in Nigeria, Tanzania, and Ghana SPARCo registries.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">bioRxiv</abbr></div> <div id="gregorova_early_2024" class="col-sm-8"> <div class="title">Early NK-cell and T-cell dysfunction marks progression to severe dengue in patients with obesity and healthy weight</div> <div class="author"> Michaela Gregorova, Marianna Santopaolo, Lucy C. Garner, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Divya Diamond, Narayan Ramamurthy, Tran Thuy Vi, Nguyet Minh Nguyen, Nguyen Lam Vuong, Eben Jones, Mike Nsubuga, Curtis Luscombe, Hoa Vo Thi My, Ho Quang Chanh, Nguyen Thi Xuan Chau, Dong Thi Hoai Tam, Duyen Huynh Thi Le, Cao Thi Tam, Paul Klenerman, Sophie Yacoub, Laura Rivino' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em></em> Sep 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2024.09.06.611687"></span> <span class="__dimensions_badge_embed__" data-doi="10.1101/2024.09.06.611687" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Dengue is a mosquito-borne virus infection affecting half of the world’s population for which therapies are lacking. The role of T and NK-cells in protection/immunopathogenesis remains unclear for dengue. We performed a longitudinal phenotypic, functional and transcriptional analyses of T and NK-cells in 124 dengue patients using flow cytometry and single-cell RNA-sequencing. We show that T/NK-cell signatures early in infection discriminate patients who will progress to severe dengue (SD) from those who do not. In patients with overweight/obesity these signatures are exacerbated compared to healthy weight patients, supporting their increased susceptibility to SD. In SD, CD4+/CD8+ T-cells and NK-cells display increased co-inhibitory receptor expression and decreased cytotoxic capacity compared to non-SD. Furthermore, type-I Interferon signalling is downregulated in SD, suggesting defective virus-sensing mechanisms may underlie NK/T-cell dysfunction. We propose that dysfunctional “professional killer” T/NK-cells underpin dengue pathogenesis. Our findings pave the way for the evaluation of immunomodulatory therapies for dengue.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="nsubuga_machine_2023" class="col-sm-8"> <div class="title">A machine learning approach to predict E. coli antibacterial resistance using whole-genome sequencing data</div> <div class="author"> Mike Nsubuga</div> <div class="periodical"> Sep 2023 </div> <div class="periodical"> Accepted: 2024-02-28T08:06:49Z </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Background: Antimicrobial resistance (AMR) is a significant global health threat, particularly impacting low- and middle-income countries(LMICS) such as Uganda, where reliable and rapid methods for detecting AMR in E. coli and other pathogens are scarce. This lack can lead to inappropriate treatment and the spread of drug-resistant infections. This thesis undertakes a comprehensive study, where various machine learning models to predict AMR in E. coli for ciprofloxacin(CIP), ampicillin(AMP), and cefotaxime(CTX) were trained on whole genome sequencing (WGS) data from England where such data is more readily available. A separate Ugandan dataset was used for validation purposes, further demonstrating the generalizability and effectiveness of the models in LMICS. Methods: 1496 (CIP), 1428 (CTX), and 1396 (AMP) sequences from England were divided into training and testing. 42 from Uganda were used for validation. Eight different machine learning models were trained and tested: Logistic Regression(LR), Random Forest(RF), Gradient Boosting(GB), XGBoost(XGB), LightGBM(LGBM), CatBoost(CB), Feed-Forward Neural Network(FFNN), and Support Vector Machine(SVM). The models were evaluated based on precision, recall, and Area Under the Receiver Operating Characteristic Curve (AUC-ROC). Upsampling techniques were implemented to address class imbalance in the data. Results: Model predictive performance varied significantly across different antibiotics, underlining the critical role of model selection and dataset characteristics. Notably, the FFNN model demonstrated superior performance during testing for CIP (accuracy 84%; F1 0.55; AUC 91%), LR for CTX (accuracy 91%; F1 0.37; AUC 83%) and GB for AMP (accuracy 57%; F1 0.62, AUC 53%), while the LGBM and RF models outperformed others in same scenarios (p \textless 0.001). Upsampling did not significantly improve the models’ performance, underscoring the complexity and high-dimensionality of SNP data. Despite high accuracy scores with the Ugandan validation dataset(FFNN with CIP accuracy 95%, LR with AMP accuracy 98% and GB with CTX accuracy 65%), the models struggled with the recall metric due to severe class imbalance. Key mutations associated with antimicrobial resistance were identified for these antibiotics. Conclusion: As the threat of AMR continues to rise, the successful application of these models - particularly on the Ugandan dataset, signals a promising avenue for improving AMR detection and treatment strategies in LMICS were genomic data is scarce. This work thus not only expands our current understanding of the genetic underpinnings of AMR but also provides a robust methodological framework that can guide future research and applications in the fight against antimicrobial resistance.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BMC</abbr></div> <div id="buyego_feasibility_2022" class="col-sm-8"> <div class="title">Feasibility of virtual reality based training for optimising COVID-19 case handling in Uganda</div> <div class="author"> Paul Buyego, Elizabeth Katwesigye, Grace Kebirungi, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Mike Nsubuga, Shirley Nakyejwe, Phillip Cruz, Meghan C. McCarthy, Darrell Hurt, Andrew Kambugu, Joseph Walter Arinaitwe, Umaru Ssekabira, Daudi Jjingo' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>BMC Medical Education</em>, Apr 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1186/s12909-022-03294-x"></span> <span class="__dimensions_badge_embed__" data-doi="10.1186/s12909-022-03294-x" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Epidemics and pandemics are causing high morbidity and mortality on a still-evolving scale exemplified by the COVID-19 pandemic. Infection prevention and control (IPC) training for frontline health workers is thus essential. However, classroom or hospital ward-based training portends an infection risk due to the in-person interaction of participants. We explored the use of Virtual Reality (VR) simulations for frontline health worker training since it trains participants without exposing them to infections that would arise from in-person training. It does away with the requirement for expensive personal protective equipment (PPE) that has been in acute shortage and improves learning, retention, and recall. This represents the first attempt in deploying VR-based pedagogy in a Ugandan medical education context.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2025 Mike Nsubuga. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>